roles/openstack-setup/tasks/users.yml

```diff
@@ -12,5 +12,5 @@
                     login_user=provider_admin
                     login_password={{ secrets.provider_admin_password }}
   with_items: keystone.services
-  when: endpoints[item.name] is defined and endpoints[item.name]
+  when: "{{item.name}}.enabled is undefined or {{item.name}}.enabled|bool"
```

Comment:
This seems to change the logic a bit, or at least change where the logic is sourced from. At least I'd want some commentary around this.